catroid/src/main/java/org/catrobat/catroid/common/ScreenValues.java

```diff
@@ -31,6 +31,9 @@
 	public static int SCREEN_WIDTH;
 	public static int SCREEN_HEIGHT;
 
+	public static final int CAST_SCREEN_WIDTH = 1280;
```

Comment:
could you please check if it's possible to get these values from the device, in order to support other aspect ratios and/or resolutions.